classes/class-p4-control-panel.php

```diff
@@ -247,28 +253,15 @@ public function check_search_indexer() {
 				$cp_action = filter_input( INPUT_GET, 'cp-action', FILTER_SANITIZE_STRING );
 
 				// CSRF check and action check.
-				if ( wp_verify_nonce( $cp_nonce, 'cp-action' ) && 'check_search_indexer' === $cp_action ) {
-					$threshold = 180;   // Period in seconds over which the Indexer will be considered stalled.
-					$response  = [];
-
-					$last_activity = searchwp_get_setting( 'last_activity', 'stats' );
-					$running       = searchwp_get_setting( 'running' );
-					$doing_delta   = searchwp_get_option( 'doing_delta' );
-					$busy          = searchwp_get_option( 'busy' );
-
-					if ( ! is_null( $last_activity ) && false !== $last_activity ) {
-						// If more than $threshold seconds have passed and the Indexer
-						// is not running or doing_delta or busy then consider it stalled.
-						if (
-							( current_time( 'timestamp' ) > $last_activity + absint( $threshold ) )
-							&& ! $running && ! $doing_delta && ! $busy
-						) {
-							$response['message'] = __( 'Indexer has stalled', 'planet4-master-theme-backend' );
-							$response['class']   = 'cp-error';
-						} else {
-							$response['message'] = __( 'Indexer is awake', 'planet4-master-theme-backend' );
-							$response['class']   = 'cp-success';
-						}
+				if ( wp_verify_nonce( $cp_nonce, 'cp-action' ) && 'check_elasticsearch' === $cp_action ) {
+					$response = [];
+
+					if ( ep_elasticsearch_can_connect() ) {
```

Comment:
Looks good! This does not seem to work with latest elasticpress version, or is it happening only for me?

Reply:
The "Check Elasticsearch" link is failing on my local as well - 
![image](https://user-images.githubusercontent.com/5357471/60879423-3669b180-a25f-11e9-87cd-285e6bab3425.png)

Reply:
Fixed that. I had not updated to latest EP version, so it was working with 2.8.*

Reply:
![image](https://user-images.githubusercontent.com/14092452/60888699-1c7c9e80-a260-11e9-8e25-7f43ac93c6d7.png)